Metro-Records/la-metro-councilmatic
# Install Wagtail and transition About page to CMS

## Overview

See title.

An initial draft of the CMS user guide can be found here: https://docs.google.com/document/d/1EbNvf_KPNvRILAWdkM5uDLFECssfPQP8p1KN6Nv_nj4/edit?usp=sharing

- Addresses #1100
- Connects #1168, #885 

### Demo

<img width="1512" alt="Screenshot 2024-11-21 at 8 31 25 AM" src="https://github.com/user-attachments/assets/d37bf30d-5497-41a1-aaec-2662e1c0e6ff">

### Notes

The following steps are planned after this PR, not necessarily in this order:

- Migrate alert management to the CMS
- Create pattern for editing content on pages with some data elements, e.g., Committees, Board Members
  - Migrate management of board member bios and headshots
- Create pattern for editing content that appears across a set of detail pages, e.g., public comment instructions
  - CMS will not enable customizing information about particular events / board reports – that information should be entered into Legistar 
- Create pattern for managing navigation groupings
- Create pattern for creating new arbitrary pages

## Testing Instructions

- View the About Us page on the review app and confirm it looks and works good.
- Log into the CMS for the review app (`/cms`) using the credentials provided in Slack.
- Make some edits to the About Us page and confirm they show up as expected, etc.
